root.tf

```diff
@@ -484,6 +493,36 @@ module "export_step_function" {
   })
 }
 
+module "new_export_bucket" {
+  source            = "./da-terraform-modules/s3"
+  bucket_name       = "tdr-export-${local.environment}"
+  kms_key_arn       = module.s3_external_kms_key.kms_key_arn
+  common_tags       = local.common_tags
+  use_random_suffix = true
+}
+
+module "new_export_bucket_judgment" {
```

Comment:
Maybe rename the module something a bit more meaningful such as: `flat_format_export_bucket_judgment`